test/test_browser.py

```diff
@@ -5290,6 +5290,9 @@ def test(args):
     'main_thread': (['-sPTHREAD_POOL_SIZE=5'],),
     # using proxy_to_pthread also works, of course
     'proxy_to_pthread': (['-sPROXY_TO_PTHREAD', '-sINITIAL_MEMORY=32MB', '-DPROXYING'],),
+    # using BigInt support affects the ABI, and should not break things. (this
+    # could be tested on either thread; do the main thread for simplicity)
+    'bigint': (['-sPTHREAD_POOL_SIZE=5', '-sWASM_BIGINT'],),
```

Comment:
Can we modify `test/other/test_parseTools.js` to test this fix more directly/precisely?

Reply:
Added a test there now.